### PR TITLE
feat(replays): Add video entry to DataCategory enum

### PR DESCRIPTION
**Pull request by cmanallen**
During the beta period we're not emitting accepted replay outcomes for mobile replays (to prevent customer's from being billed).  Tracking usage through Looker is difficult without this data.  We want to emit a non-billed outcome in our consumer for tracking purposes.  This category is assumed to be temporary.

Related: https://github.com/getsentry/team-replay/issues/452

### CODE REVIEW

**Review by Dav1dde — CHANGES_REQUESTED** [review]
Please follow the docs right below your change:

https://github.com/getsentry/relay/blob/93144ee6d5b9fce17cf8a964c1a417e6d1111152/relay-base-schema/src/data_category.rs#L91-L96

**Comment by Dav1dde** [comment]
Ah, I forgot before. Please also add a changelog entry to `py/CHANGELOG.md` since this will be a change in the Python lib and Sentry (or consumers) will depend on that change.

**Comment by cmanallen** [comment]
@Dav1dde Any ideas?

```
(.venv) ➜  relay-cabi git:(cmanallen/replays-add-video-data-category) ✗ make header
cbindgen -c cbindgen.toml . -o include/relay.h
make: cbindgen: No such file or directory
make: *** [include/relay.h] Error 1
(.venv) ➜  relay-cabi git:(cmanallen/replays-add-video-data-category) ✗ ls
Cargo.toml    Makefile      cbindgen.toml include       src
(.venv) ➜  relay-cabi git:(cmanallen/replays-add-video-data-category) ✗ ls include
relay.h
```

**Comment by cmanallen** [comment]
@Dav1dde Oh, I don't think I have cbindgen installed.  Let's see if that fixes it.  EDIT: It did.

File: relay-quotas/src/quota.rs [hunk 1]
```diff
@@ -200,7 +200,7 @@ impl CategoryUnit {
             DataCategory::Session => Some(Self::Batched),
             DataCategory::ProfileDuration => Some(Self::Milliseconds),
 
-            DataCategory::Unknown => None,
+            DataCategory::Unknown | DataCategory::ReplayVideo => None,
```
Comment (Dav1dde) [inline relay-quotas/src/quota.rs]: This could go with the `Self::Count` block I think.

**Review by Dav1dde — APPROVED** [review]

**Pull request merged**